test/unit/http/DataQueryEndpoints.test.ts

```diff
@@ -118,6 +118,12 @@ describe('DataQueryEndpoints', () => {
                     .expect(streamMessages.map((msg) => msg.serialize(30)), done)
             })
 
+            it('responds with raw format', (done) => {
```

Comment:
Test for when `format` is given invalid value?

Reply:
Added test